lib/confirmation.rb

```diff
@@ -0,0 +1,20 @@
+class Confirmation
```

Comment:
It does two things. Verifies user cost and totals up the real cost. Maybe pull one of the responsibilities into another class?